baselines/pfsp/lib/c_bounds_gpu.cu

```diff
@@ -81,11 +81,11 @@ sum_unscheduled_gpu(const lb1_bound_data lb1_data, const int * const permutation
   const int nb_machines = lb1_data.nb_machines;
   const int * const p_times = lb1_data.p_times;
 
-  memset(remain, 0, nb_machines*sizeof(int));
-  
-  // for (int j = 0; j < nb_machines; j++) {
-  //   remain[j] = 0;
-  // }
+  //memset(remain, 0, nb_machines*sizeof(int));
```

Comment:
I encountered problems with memset function when doing hipify. He gave me an error saying he couldn't find a match for the function. But for this file I can put back the previous version of the code because I am not doing hipify for it anymore.

Reply:
No, not in particular actually. Because I saw you using memset in another vector I thought about putting it here to see if I would notice any difference. But no actually, I will come back to the previous version.